Jenkinsfile

```diff
@@ -15,28 +15,23 @@ pipeline {
         JAVA_HOME = '/opt/jdk-17/'
       }
       steps {
-        sh './mvnw -V verify checkstyle:check spotbugs:check -Dmaven.test.failure.ignore -Dcheckstyle.failOnViolation=false -Dspotbugs.failOnError=false'
+        sh './mvnw -V verify checkstyle:checkstyle spotbugs:spotbugs -Dmaven.test.failure.ignore -Dcheckstyle.failOnViolation=false -Dspotbugs.failOnError=false'
```

Comment:
What does this change do?

Reply:
We kept it this way during your previous PR, and that change enabled us to not stop the build even if there are errors.

Reply:
Thank you very much!